index.html

```diff
@@ -1016,6 +1016,19 @@ <h3>Colored buttons</h3>
             <button type="button" class="btn btn-outline-danger">Danger</button>
             <button type="button" class="btn btn-warning">Warning</button>
             <button type="button" class="btn btn-outline-warning">Warning</button>
+
+            <br><br>            
```

Comment:
You must configure your IDE to remove unwanted whitespace at the end of lines.
```suggestion
            <br><br>
```

Reply:
I added the conf and removed whitespaces